api/Document.json

```diff
@@ -7769,7 +7769,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
I was a bit skeptical of this, so tested it separate in https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8540 using IE11. But `document.origin` seems to not have been in IE, so this change is correct.